### PR TITLE
Update in settings migration to activity

### PR DESCRIPTION
**Pull request by robece**

### CODE REVIEW

File: Producer/Producer.Droid/Settings/SettingsFragment.cs [hunk 1]
```diff
@@ -18,7 +18,6 @@ public override void OnCreate (Bundle savedInstanceState)
 			FindPreference (SettingsKeys.VersionDescription).Summary = Settings.VersionDescription;
 		}
 
-
```
Comment (NateRickard) [inline Producer/Producer.Droid/Settings/SettingsFragment.cs:21]: @rcervantes we _want_ two lines between each method - don't remove them!

File: Producer/Producer.Droid/Settings/SettingsActivity.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	{
+		protected override void OnCreate (Bundle savedInstanceState)
+		{
+			base.OnCreate (savedInstanceState);
+
+			SetContentView (Resource.Layout.Settings);
+
+			FragmentManager.BeginTransaction ()
+			.Add (Resource.Id.fragment_container, new SettingsFragment ())
+			.Commit ();
```
Comment (NateRickard) [inline Producer/Producer.Droid/Settings/SettingsActivity.cs:28]: Any reason to do this in code vs using the <fragment> tag in the layout file?

File: Producer/Producer.Droid/Settings/SettingsFragment.cs (hunk 1 above)
Comment (NateRickard) [reply Producer/Producer.Droid/Settings/SettingsFragment.cs:21]: Can you remove these changes from the PR?

**Review by NateRickard — CHANGES_REQUESTED** [review]

**Review by colbylwilliams — COMMENTED** [review]
Nate's comments cover it.  I have nothing additional to add.

**Review by NateRickard — DISMISSED** [review]

**Review by NateRickard — APPROVED** [review]
Changes look good

**Pull request merged**